packages/core-antd/src/components/layout/Page.tsx

```diff
@@ -13,29 +13,35 @@ interface Props {
   theme: any;
   HeaderMainSection: FC;
   logout: () => AnyAction;
+  context: any
 }
 
 const { Content } = Layout;
 
 /* eslint-disable complexity */
 export const Page = (props: Props) => {
-  const { HeaderMainSection, theme, logout, children } = props;
+  const { HeaderMainSection, theme, logout, children, context } = props;
   const router = useRouter();
-  const appConfig = useContext(AppContext).getAppConfig();
```

Comment:
` const context = useContext(AppContext)`  does **not** work here.
And when you ctrl+click on AppContext to enter the implementation, you go to `packages\core-antd\node_modules\@onr\core\src\components\App.tsx`

Maybe the path is the reason, it's importing the wrong class. I don't know why it's doing this though.

Reply:
I assume the problem is related to this Lerna thingy 😄

Reply:
Good point! I think it's very possible that it's using wrong class or instance with same name. That gives me a direction. Thanks!